tests/unit/test_ast.py

```diff
@@ -265,49 +278,126 @@ def test_compare_empty(self, type):
         assert ctx.literals["l"] == d["l"]
         assert ctx.literals["r"] == ast.Empty()
 
-    def test_contains_invalid(self):
+    @pytest.mark.parametrize("type", ["contains", "any", "all"])
+    def test_contains_invalid(self, type):
         l = ast.Literal("l")
         r = ast.Literal("r")
-        a = ast.ContainsOperator(l, r)
-        d = {"l": 1, "r": None}
+        a = ast.ContainsOperator(type, l, r)
+        d = {"l": 1, "r": 1}
```

Comment:
Why this change?

Reply:
Because I felt like the test was misleading. When looking at it, I felt like it was invalid because of the `None` but actually, it's because `l` does not support `__contains__`, so the `None` value doesn't bring anything to the test.